### PR TITLE
[test][docker] Add Dockerfile test

### PR DESCRIPTION
**Pull request by jaylees14**
Just add a little test for our Dockerfile generator. The actual testing of Dockerfiles we output will come with the builder

### CODE REVIEW

**Review by Lewiky — APPROVED** [review]
LGTM

File: src/it/scala/temple/generate/docker/DockerGeneratorIntegrationTest.scala [hunk 1]
```diff
@@ -13,4 +15,23 @@ class DockerGeneratorIntegrationTest extends HadolintSpec with Matchers with Bef
...
+        Env("alpha", "beta"),
+        Add("http://example.com/big.tar.xz", "/usr/src/things/"),
+        Copy("/tmp/foo", "/tmp/bar"),
+        Entrypoint("myapp", Seq("--build", "/path/to/random/file")),
+        Volume("myvolume"),
+        WorkDir("/etc/bin/lib/dev/rand"),
+        Expose(8080),
+      ),
+    )
+    validate(DockerfileGenerator.generate(content)) shouldBe empty
```
Comment (c-harding) [inline src/it/scala/temple/generate/docker/DockerGeneratorIntegrationTest.scala]: Is it worth checking the string value too?

Comment (c-harding) [reply src/it/scala/temple/generate/docker/DockerGeneratorIntegrationTest.scala]:
```suggestion
    val validationErrors = validate(DockerfileGenerator.generate(content))
    validationErrors shouldBe empty
```

**Review by c-harding — APPROVED** [review]
LVG2M

**Pull request merged**